.azure/pipelines/ci.yml

```diff
@@ -130,705 +150,753 @@ variables:
   - ${{ if in(variables['Build.Reason'], 'PullRequest') }}:
     - name: _SignType
       value: test
+- name: runCodeQL3000
+  value: ${{ or(eq(variables['Build.Reason'], 'Schedule'), and(eq(variables['Build.Reason'], 'Manual'), eq(parameters.runCodeQL3000, 'true'))) }}
 
 stages:
 - stage: build
   displayName: Build
   jobs:
-  # Code check
-  - ${{ if or(eq(variables['System.TeamProject'], 'public'), in(variables['Build.Reason'], 'PullRequest', 'Manual')) }}:
+  - ${{ if and(ne(variables['System.TeamProject'], 'public'), eq(variables.runCodeQL3000, 'true')) }}:
     - template: jobs/default-build.yml
       parameters:
-        jobName: Code_check
-        jobDisplayName: Code check
+        jobName: build
+        jobDisplayName: Build and run CodeQL3000
         agentOs: Windows
+        codeSign: false
+        # Component governance and SBOM creation are not needed here. Disable what Arcade would inject.
+        disableComponentGovernance: true
```

Comment:
/fyi neither this parameter nor the `skipComponentGovernanceDetection` variable make what they do obvious in their names. `disableComponentGovernance` is used in eng/common/job/job.yml to control Arcade's CG task insertion. `skipComponentGovernanceDetection` controls the auto-injection; its evaluation shows up under `Job preparation parameters` in the first console log of each job (above `Initialize job`).

When both are `false` (as usually happens in our official builds), we see both `ComponentGovernanceComponentDetection` and `Component Detection (auto-injected by policy)` build steps in the job. The second does nothing because the first sets the `CG_RAN` variable. Not useful to have the second at all (disabled in a change below).